templates/properties/ldap-healthcheck-config.json.erb

```diff
@@ -0,0 +1,46 @@
+{
+   "ldap-config" : {
+<% if ldap_server -%>
+     "server" : <%=ldap_server%>,
+<% end -%>
+<% if ldap_port -%>
+     "port" : <%=ldap_port%>,
+<% end -%>
+<% if ldap_serach_base -%>
+     "serach_base" : <%=ldap_serach_base%>,
+<% end -%>
+<% if ldap_username -%>
+     "username" : <%=ldap_username%>,
+<% end -%>
+<% if ldap_password -%>
+     "password" : <%=ldap_password%>,
+<% end -%>
+<% if ldap_pool_size -%>
+     "pool_size" : <%=ldap_pool_size%>,
+<% end -%>
+<% if ldap_use_tls -%>
+     "use_tls" : <%=ldap_use_tls%>,
+<% end -%>
+<% if truststore_url -%>
+     "truststore" : <%=truststore_url%>,
+<% end -%>
+<% if truststore_password -%>
+     "truststore_password" : <%=truststore_password%>,
+<% end -%>
+<% if connectionTimeoutMS -%>
+     "connectionTimeoutMS" : <%=connectionTimeoutMS%>,
+<% end -%>
+<% if responseTimeoutMS -%>
+     "responseTimeoutMS" : <%=responseTimeoutMS%>,
+<% end -%>
+<% if debug -%>
+     "debug" : <%=debug%>,
+<% end -%>
+<% if keepAlive -%>
+     "keepAlive" : <%=keepAlive%>,
+<% end -%>
+<% if poolMaxConnectionAgeMS -%>
+     "poolMaxConnectionAgeMS" : <%=poolMaxConnectionAgeMS%>,
```

Comment:
The , at the end here would result in invalid JSON, so you'll either just need to remove this last one (probably okay), or account for the fact that any one of these could be the last one if the following ones are not populated.